### ISSUE TITLE
Release 0.9.3

### ISSUE THREAD

**Issue by savente93** [Deltares/hydromt#740]
### Kind of request

Adding new functionality

### Enhancement Description

There are some functionalities that people have been asking for so it would be good to do another release soon. Please add issues for things you'd like to include for this release below: 
- [x] #739   

### Additional Context

@hboisgon @DirkEilander @Jaapel @Tjalling-dejong please add issues you'd like included

**Comment by DirkEilander** [comment]
- check with @RubenImhoff if the issue raised on the Teams chat has been resolved. If not a reproducible example is required to investigate it further.

**Comment by RubenImhoff** [comment]
Which issue specifically, @DirkEilander?

**Comment by hboisgon** [comment]
Issue that you could not read merit_hydo_ihu or merit_hydro:

> As I had the same issue, I've updated hydromt and hydromt-wflow, but I run into a new issue:
2024-01-11 11:50:55,364 - build - rasterdataset - INFO - Reading merit_hydro_ihu raster data from /mnt/proj3/dd-23-93/hydromt/topography/merit_hydro_ihu/30sec/*.tif
2024-01-11 11:50:58,967 - build - main - ERROR - '/vsimem/fa04f0fc40154aebab173ab32e410a1d' not recognized as a supported file format.
Traceback (most recent call last):
  File "fiona/ogrext.pyx", line 136, in fiona.ogrext.gdal_open_vector
  File "fiona/_err.pyx", line 291, in fiona._err.exc_wrap_pointer
fiona._err.CPLE_OpenFailedError: '/vsimem/fa04f0fc40154aebab173ab32e410a1d' not recognized as a supported file format.
Seem to be related to Fiona at first, but after updating Fiona, it still gives the same error. Anyone familiar with this error?

**Comment by RubenImhoff** [comment]
It does not seem to solve the issue for me.. I updated the environment with a `pip install git+https://github.com/Deltares/hydromt.git`. That should be sufficient, as everything has been committed, right?

**Comment by DirkEilander** [comment]
Can you share a minimal reproducible example in a new issue? I.e., does the error also occur when just loading the dataset from the catalog with `DataCatalog.get_rasterdataset('merit_hydro_ihu')` or only in the context of building a Wflow model? If you are not using the deltares data, can you add you catalog file?

**Comment by RubenImhoff** [comment]
Through python, I can load the dataset with `DataCatalog.get_rasterdataset('merit_hydro_ihu')`. But when building a model, it fails. Why is a mystery to me. I'll make an issue.

**Comment by RubenImhoff** [comment]
See issue #742. Yes, it goes through the command line interface with the same python installation (same environment is started prior to calling the command line).

**Comment by hboisgon** [comment]
Also fix #744 before release

**Comment by savente93** [comment]
still waiting on #747 and #751 to be merged for the new release

**Comment by DirkEilander** [comment]
Also #758 should be merged before releasing

**Comment by DirkEilander** [comment]
I just fixed a small bug based in #761 which is ready to be reviewed too
